### PR TITLE
Create API for registration of fence gate.

### PR DESCRIPTION
**Pull request by ghost**
screenshot_20160313_215112
And, add 4 fence gates.

### CODE REVIEW

**Comment by kilbith** [comment]
Don't you see you're using the wrong texture files ?

A minimum of rigor in your work, please.

**Comment by Rui914** [comment]
Oh!
I never noticed that.

I'll fix soon.

**Comment by Rui914** [comment]
I fixed it.
I am an idiot...

**Comment by Ekdohibs** [comment]
:+1:

**Comment by sofar** [comment]
Fine by me. Maybe for post-0.4.14? Keep something for later :)

**Comment by Thomas--S** [comment]
I suggest to add this to 0.4.14, so that mods can start using the API soon.

**Comment by paramat** [comment]
i'll review, would be good to have it for release.

File: mods/doors/init.lua [hunk 1]
```diff
@@ -651,59 +651,99 @@ minetest.register_craft({
...
+
+doors.register_fencegate("doors:gate_aspen_wood", {
+	description = "Aspen Fence Gate",
+	texture = "default_aspen_wood.png",
+	material = "default:aspen_wood",
+	groups = {choppy = 2, oddly_breakable_by_hand = 2, flammable = 2}
 })
 
+minetest.register_alias("doors:gate_closed", "doors:gate_wood_closed")
+minetest.register_alias("doors:gate_open", "doors:gate_wood_open")
```
Comment (sofar) [inline mods/doors/init.lua]: Remove these 2 aliases please. This nodetype wasn't introduced in a previously released build, so maintaining backwards compatible code for it is somewhat unneeded.

**Comment by sofar** [comment]
Remove the alias lines, then it's good to go. There's no need to have an alias for a node that's been in development code for only a week (barely).

**Comment by Rui914** [comment]
I removed the alias lines.
Also I added sounds (see: #937).

**Comment by sofar** [comment]
Ahh, excellent, thanks for picking up the sounds as well.

**Comment by paramat** [comment]
:+1:

**Comment by paramat** [comment]
a2ecc51fbc75dd0411c6bbe096f8c39b3a6abc6c

**Pull request closed without merging**